ci/simple_config.toml

```diff
@@ -0,0 +1,25 @@
+[global]
+timeout = '10s'
+strategy = 'naive'
+
+[[chains]]
+id = 'ibc0'
```

Comment:
Nit: It'd be best to have `ibc-0` and `ibc-1` as chain names, to be consistent with the rest of our docs.

Reply:
OK, makes sense. I'll change all the logic that uses `ibc0` and `ibc1` to `ibc-0` and `ibc-1`

Reply:
Renamed chain names to make it consistent => https://github.com/informalsystems/ibc-rs/commit/2cc90a2025776abb37d303db5705f03697a65b5b